pkg/kapis/iam/v1alpha2/handler.go

```diff
@@ -65,6 +65,12 @@ type GroupMember struct {
 	GroupName string `json:"groupName"`
 }
 
+type GroupRoleBinding struct {
+	GroupName string `json:"groupName"`
+	RoleName  string `json:"roleName"`
+	Namespace string `json:"namespace,omitempty"`
+}
```

Comment:
Why not use rolebinding directly, the subject can be group kind.

Reply:
Refined. Use rolebinding and workspacerolebinding directly.